src/support/azext_support/custom.py

```diff
@@ -7,16 +7,42 @@
 # pylint: disable=too-many-locals
 
 import json
+import base64
+import os
+import math
+from azext_support._validators import validate_file_path, validate_file_size
+from azext_support._validators import validate_file_extension, validate_file_name
+
 from datetime import date, datetime, timedelta
 
-from azext_support._utils import (get_bearer_token, is_quota_ticket,
-                                  is_technical_ticket, parse_support_area_path)
+from azext_support._utils import (
+    get_bearer_token,
+    is_quota_ticket,
+    is_technical_ticket,
+    parse_support_area_path,
+)
 from knack.log import get_logger
 
 from .aaz.latest.support.in_subscription.tickets import Update as _Update
 from .aaz.latest.support.in_subscription.tickets import Create as _CreateTicket
-from .aaz.latest.support.in_subscription.communication import Create as _CreateCommunication
-from .aaz.latest.support.no_subscription.communication import Create as _CreateNoSubscriptionCommunication
+from .aaz.latest.support.in_subscription.communication import (
+    Create as _CreateCommunication,
+)
+from .aaz.latest.support.no_subscription.communication import (
```

Comment:
Do we need this "(" after import? Did style check complain about it? I dont see it added for tickets